examples/lifecycleDemo.html

```diff
@@ -136,40 +188,76 @@ <h1>Lifecycle Events Log:</h1>
                     // Simulating bootstrap delay of 500ms
                     window.setTimeout(() => {
                         myClientApp.lifecycle.bootstrapped();
-
-                        myClientApp.alerting.showToastPopup(
-                            lifecycleStatusMessageTitle,
-                            'Bootstrap Complete (500ms delay)', {
-                                id: lifecycleStatusMessageId,
-                                type: 'success'
-                            }
-                        );
+                        if (requireManualToastDismissal) {
+                            myClientApp.alerting.showToastPopup(
+                                lifecycleStatusMessageTitle,
+                                `Bootstrap Complete (500ms delay):lifecycleStatusMessageId-bootstrap-${count}`, {
+                                    id: `lifecycleStatusMessageId-bootstrap-${count}`,
+                                    type: 'success',
+                                    showCloseButton: true,
+                                    timeout: 0
+                                }
+                            );
+                            count++;
+                        } else {
+                            myClientApp.alerting.showToastPopup(
+                                lifecycleStatusMessageTitle,
+                                `Bootstrap Complete (500ms delay)`, {
+                                    id: `lifecycleStatusMessageId`,
```

Comment:
^ this uses the string and not the variable, so the result is an id with the name of the variable vs 'lifecycleDemo-statusMsg'